tests/base/parameterframe/test_parameterframe.py

```diff
@@ -182,6 +183,43 @@ def test_update_values_edits_frames_values(self):
         assert frame.age.value == 30
         assert frame.age.source != "a test"
 
+    @pytest.mark.parametrize("head_keys", [None, ["name", "value"]])
+    def test_tabulator_method_formatting(self, head_keys):
```

Comment:
It's not so easy to understand what this test is doing. Would it be simpler to split this into two tests; one for each case? Then (I think) it's just a case of extracting the args/kwargs and directly asserting what we expect them to be, which should be much easier to parse.

It will be less DRY, but, in my view, tests should be specific and explicit, even if it means repeating yourself a bit.

Reply:
Sure :) I did have fun condensing that one :laughing:

Reply:
Hopefully thats easier to understand